x-pack/plugins/profiling/server/lib/setup/types.ts

```diff
@@ -23,4 +24,5 @@ export interface ProfilingSetupStepFactoryOptions {
   logger: Logger;
   spaceId: string;
   isCloudEnabled: boolean;
+  config: ProfilingConfig['config'];
```

Comment:
Assuming we move `xpack.profiling.config.symbolizer` to `xpack.profiling.symbolizer` as suggested above, we would likely have to pass `ProfilingConfig` around instead.